doc/classes/MeshInstance3D.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="MeshInstance3D" inherits="GeometryInstance3D" version="4.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		Node that instances meshes into a scenario.
+		Node that instantiates meshes into a scenario.
```

Comment:
The changes in this file should be reverted, "instance" is the correct term here